docker/cytoscape/DockerFile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:20.04
```

Comment:
How much of this is derived from the example Cytoscape Dockerfile?  Should add a comment with attribution and information in a readme about the source?

Please rename to `Dockerfile`.

Reply:
Should attribution be done in the Dockerfile, or in a readme in the docker directory?

Reply:
This is the repository that the Dockerfile is bases off of: https://github.com/cytoscape/docker-cytoscape-desktop

Reply:
Typically I would provide attribution in both.  In the readme, we should link back to the original repository and specify the license the copied materials are available under (not applicable here, but applicable for SPRAS or other code).  Then in the source file (Dockerfile or .py file) we should have a comment with a note that file was derived from <link> and again list the license terms if applicable.

For an example, see:
- https://github.com/gitter-lab/nn4dms#external-sources
- https://github.com/gitter-lab/nn4dms/blob/master/code/my_pipgcn.py